app/models/user.rb

```diff
@@ -41,5 +40,21 @@ def set_default_role
   has_one :profile
   has_many :application_letters
   has_many :requests
-       
+
+  # Returns the number of previously accepted applications from the user
+  #
+  # @param none
+  # @return [Int] of number of accepted applications
```

Comment:
As discussed on Slack, the currently displayed camp does not count, but temporary acceptance / rejection of other camps can count